composer.json

```diff
@@ -38,7 +38,7 @@
 		"docs": "https://github.com/DataValues/Geo/#usage"
 	},
 	"require": {
-		"php": ">=7.4",
+		"php": ">=8.0",
 		"data-values/data-values": "^3.0|^2.0|^1.0|~0.1",
 		"data-values/interfaces": "^1.0.0|^0.2.0",
```

Comment:
I think this should’ve been bumped to `^1.2.0`? We can’t be compatible with older data-values versions if we `withDefaultOption()` to be available unconditionally. (Noticed in [this CI build](https://integration.wikimedia.org/ci/job/quibble-composer-mysql-php74-noselenium/11415/console) for [this Wikibase change](https://gerrit.wikimedia.org/r/c/mediawiki/extensions/Wikibase/+/1115452); I’ll see how I can make that change pull in the newer data-values version.)